addons/account/static/src/components/account_move_form/account_move_form.js

```diff
@@ -6,6 +6,28 @@ import { Notebook } from "@web/core/notebook/notebook";
 import { formView } from "@web/views/form/form_view";
 import { FormCompiler } from "@web/views/form/form_compiler";
 import { FormRenderer } from "@web/views/form/form_renderer";
+import { FormController } from '@web/views/form/form_controller';
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { useService } from "@web/core/utils/hooks";
+
+
+export class AccountMoveController extends FormController {
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+    }
+
+    async deleteRecord() {
+        const move_id = this.model.root.resId;
+        this.is_move_end_of_chain = await this.orm.call('account.move', 'check_move_sequence_chain', [move_id]);
+        if (!this.is_move_end_of_chain) {
+            ConfirmationDialog.template = "account.AccountMoveConfirmationDialog";
```

Comment:
You shouldn't change the template like this.
Try to archive a customer after getting your modified dialog